### PR TITLE
Stop odh model controller from adding/removing namespace labels

### PR DESCRIPTION
**Pull request by anishasthana**
Test image: quay.io/anishasthana/odh-model-controller-test:v9

Signed-off-by: Anish Asthana <anishasthana1@gmail.com>

### CODE REVIEW

File: odh-model-controller/controllers/inferenceservice_controller.go [hunk 1]
```diff
@@ -74,11 +74,6 @@ func (r *OpenshiftInferenceServiceReconciler) Reconcile(ctx context.Context, req
 		return ctrl.Result{}, err
 	}
 
-	err = r.ReconcileNamespace(inferenceservice, ctx)
```
Comment (HumairAK) [inline odh-model-controller/controllers/inferenceservice_controller.go:77]: Any reason to keep [inferenceservice_namespace.go](https://github.com/opendatahub-io/modelmesh-serving/blob/main/odh-model-controller/controllers/inferenceservice_namespace.go) around if this is being removed? This seems like its only usage.

File: quickstart/quickstart.sh [hunk 2]
```diff
@@ -33,6 +33,7 @@ oc new-project $MODELMESH_PROJECT
 oc apply -f ../manifests/kfdef.yaml -n $MODELMESH_PROJECT
 
 oc new-project $INFERENCE_SERVICE_PROJECT
+oc label namespace $INFERENCE_SERVICE_PROJECT "modelmesh-enabled=true" --overwrite=true || echo "INFO: modelmesh-enabled label already exists."
```
Comment (HumairAK) [inline quickstart/quickstart.sh:36]: > echo "INFO: modelmesh-enabled label already exists." 

That is not the only reason an error could occur (e.g. permission issues), I would use a broader message like "failed to apply label"

File: odh-model-controller/controllers/inferenceservice_controller.go (hunk 1 above)
Comment (anishasthana) [reply odh-model-controller/controllers/inferenceservice_controller.go:77]: I did it remove it as part of the PR:

File: quickstart/quickstart.sh (hunk 2 above)
Comment (anishasthana) [reply quickstart/quickstart.sh]: Fixed :-)

Comment (anishasthana) [reply quickstart/quickstart.sh]: Fixed :-)

**Review by DaoDaoNoCode — APPROVED** [review]
lgtm, thanks!

**Comment by HumairAK** [comment]
Tested PR, label does not get applied, and unless you manually apply it, modelmesh will complain: 

```
{"level":"info","ts":1667858267.6023016,"logger":"controllers.ServingRuntime","msg":"Runtime is disabled, incompatible with modelmesh, or namespace is not modelmesh-enabled","servingruntime":"mesh-test/ovms-1.x"}
```
As expected, and once label is manually applied, runtime pods are deployed.

**Pull request merged**